### PR TITLE
Portfolio's Desktop Version

### PR DESCRIPTION
**Pull request by zainsadaqat**
> Portfolio's Desktop Version contains the following sections:
* Navbar ( Developed with Flexbox ) 
* Banner or Hero ( Developed with Flexbox )
* Work ( Developed with CSS Grid and Flexbox )
* About ( Developed with Flexbox )
* Contact or Footer ( Developed with Flexbox )

> Note: 
I'd really appreciate it if you highlight my code that is not clean and semantic Because I'm trying to write clean and semantic code as much as possible.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -324,3 +331,122 @@ body {
...
+    padding: 100px 0;
+  }
+
+  .about-container > .proficiency-section {
+    flex: 1 1 60%;
+    padding: 100px 0;
+  }
+
+  .languages-container {
+    display: flex;
```
Comment (Lameck1) [inline style.css:429]: - [x] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

File: style.css [hunk 2]
```diff
@@ -324,3 +331,122 @@ body {
...
+    text-align: center;
+    margin-left: 32px;
+    color: #344563;
+  }
+
+  .hero-section {
+    background-image: url(/images/desktop-background.svg);
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
```
Comment (Lameck1) [inline style.css]: - [x] [OPTIONAL] There is nothing bad with how you have implemented this background or any other. However, using shorthand properties is fun and can save you lines of CSS code. Let me share with you this [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties) about shorthand properties, you can go through it at your own convenience.

**Review by Lameck1 — APPROVED** [review]
Hi, @zainsadaqat,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (zainsadaqat) [reply style.css:429]: Alright man

**Comment by zainsadaqat** [comment]
# Thanks, @Lameck1 for approving this pull request!

**Pull request merged**